client/shaders/nodes_shader/opengl_fragment.glsl

```diff
@@ -188,10 +189,9 @@ float getPenumbraRadius(sampler2D shadowsampler, vec2 smTexCoord, float realDist
 		}
 	}
 
-	depth *= 1e0 * world_to_texture / max(n, 1.0);
-
+	depth *= world_to_texture / max(n, 1);
```

Comment:
Getting this now:
ERROR[Main]: Irrlicht: 0(241) : error C1101: ambiguous overloaded function reference "max(int, int)"

Don't you just love shaders? :(

Reply:
```
ERROR[Main]: Irrlicht: 0(241) : error C1101: ambiguous overloaded function reference "max(int, int)"
(0) : gp5 cp50 fp50 vp50 gp50 cpf fpf vpf gpf superp float max(superp float, superp float)
(0) : gp5 mediump float max(mediump float, int)
(0) : ps gp5 gp4 cp50 fp50 vp50 gp50 cpf fpf vpf gpf lowp float max(lowp float, lowp float)
(0) : ps gp5 gp4 cp50 fp50 vp50 gp50 cpf fpf vpf gpf mediump float max(mediump float, mediump float)
(0) : float max(float, float)
```

Reply:
Fixed in the latest commits.